src/types/index.ts

```diff
@@ -895,6 +895,10 @@ export const UnifiedConfigSchema = TestSuiteConfigSchema.extend({
 
 export type UnifiedConfig = z.infer<typeof UnifiedConfigSchema>;
 
+export enum ConfigType {
```

Comment:
sorry after thinking about this, I would prefer if we did not define a type for it and just made the resolveConfigs new argument of type const or string union. There's no good reason to expose this to the package and if we were to define config type it should probably also include redteam